napari/_vispy/vispy_scale_bar_visual.py

```diff
@@ -117,57 +145,42 @@ def _on_data_change(self, event):
         self.text_node.color = color
 
     def _on_visible_change(self, event):
-        """Change visibiliy of scale bar."""
+        """Change visibility of scale bar."""
         self.node.visible = self._viewer.scale_bar.visible
         self.text_node.visible = self._viewer.scale_bar.visible
         self._on_zoom_change(None)
 
-    def _on_position_change(self, event):
+    def _on_text_change(self, event):
+        """Update text information"""
+        self.text_node.font_size = self._viewer.scale_bar.font_size
+
+    def _on_position_change(self, _evt=None):
```

Comment:
Curious about `event` vs `_evt` I would suggest we always used `event`, as it makes it more clear?

Reply:
I'd also prevent `event`, it's what we have used most places